src/main/kotlin/team/themoment/gsmNetworking/global/security/SecurityConfig.kt

```diff
@@ -64,6 +64,16 @@ class SecurityConfig(
             .mvcMatchers(HttpMethod.POST, "/api/v1/mentor").hasAnyRole(
                 Authority.TEMP_USER.name
             )
+            // /tempMentor
+            .mvcMatchers(HttpMethod.GET, "/api/v1/temp-mentor/*").hasAnyRole(
+                Authority.TEMP_USER.name
+            )
+            .mvcMatchers(HttpMethod.GET, "/api/v1/temp-mentor/").hasAnyRole(
+                Authority.TEMP_USER.name,
+            )
```

Comment:
mvcMatchers 를 사용했으니, 아래와 같이 수정해보는게 어떤가요?
```suggestion
            // /tempMentor
            .mvcMatchers(HttpMethod.GET, "/api/v1/temp-mentor/**").hasAnyRole(
                Authority.TEMP_USER.name
            )
```

Reply:
수정하였습니다!
[34cb36c](https://github.com/themoment-team/GSM-Networking-server/pull/40/commits/34cb36c1dfd5b285c931182e7f4c346a0bc3807c)

Reply:
이 부분에 대해서는 문제 없이 잘 작동하는걸 테스트 해보셨을까요?

Reply:
넵 해보았습니다